packages/expo-camera/ios/Next/CameraViewNext.swift

```diff
@@ -636,13 +631,12 @@ public class CameraViewNext: ExpoView, EXCameraInterface, EXAppLifecycleListener
   public override func removeFromSuperview() {
     lifecycleManager?.unregisterAppLifecycleListener(self)
     super.removeFromSuperview()
+    UIDevice.current.endGeneratingDeviceOrientationNotifications()
     NotificationCenter.default.removeObserver(self, name: UIDevice.orientationDidChangeNotification, object: nil)
   }
 
   func ensureSessionConfiguration() {
-    sessionQueue.async {
-      self.updateSessionAudioIsMuted()
-    }
+    self.updateSessionAudioIsMuted()
```

Comment:
Are you sure that it's safe not to run on the session queue?

Reply:
Anywhere it's called is already putting it on the sessionQueue so I didn't think it was necessary to do it explicitly here too. I'm trying to keep as much of the configuration as sequential as possible. Moving too much onto the queue makes it very unpredictable when it's safe to start the session.

Reply:
Hmm, ok, makes sense. In this case I recommend to add doc comments for functions that must be run on the session queue, i.e. running on the proper queue must be enforced externally.

Reply:
Looking again, I can remove this function entirely.